templates/sshd_config.erb

```diff
@@ -154,12 +154,12 @@ GSSAPICleanupCredentials <%= @sshd_gssapicleanupcredentials_real %>
 UsePAM <%= @sshd_use_pam_real %>
 <% end -%>
 
-<% if @sshd_acceptenv_real == true -%>
+<% if @sshd_acceptenv_conf.length > 0 -%>
```

Comment:
We can't change this type as it would be a breaking change. Could you just revert to what you had before and change the logic in the manifest. That would have worked.

Reply:
I tried to avoid any braking change on my first version. As you want to keek less settings/attributes I can't see any reason to keep sshd_acceptenv as boolean and add one/two more attribute(s) for configuring it. Let's keep just one applying the defaults when it is not specified (and the OS supports, of course) and apply customized when it is.